lib/smart_answer_flows/student-finance-calculator/outcomes/outcome_uk_all_students.erb

```diff
@@ -19,36 +19,36 @@
 
   You could get per year:
 
-  - <%= format_money(tuition_fee_amount, pounds_only: true) %> [Tuition Fee Loan](/student-finance/parttime-students)
+  - <%= format_money(calculator.tuition_fee_amount, pounds_only: true) %> [Tuition Fee Loan](/student-finance/parttime-students)
 
   <% if calculator.maintenance_loan_amount > 0 %>
     - <%= format_money(calculator.maintenance_loan_amount, pounds_only: true) %> [Maintenance Loan](/student-finance/parttime-students)
   <% end %>
 
   ###Extra student funding
 
-  <% if all_uk_students_circumstances.include?('no') and course_studied != 'teacher-training' and course_studied != 'social-work' %>
+  <% if calculator.uk_all_circumstances.include?('no') and calculator.course_studied != 'teacher-training' and calculator.course_studied != 'social-work' %>
```

Comment:
First, I think `and` should be replaced with `&&`.

Second, this is a lot of processing for a view. Could this logic be moved to a new method in the calculator?

Reply:
Done!